cli/using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,104 @@
+[id="using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: using-roxctl-cli
+
+toc::[]
+
+[id="getting-started_{context}"]
+== Getting started
+
+The following procedure describes how to use the `roxctl central whoami` command to retrieve information about your authentication status and user profile in the Central system. The example output illustrates the data you can expect to see, including user roles, access permissions, and various administrative functions. This step allows you to review your access and roles within the Central platform.
+
+.Procedure
+
+* Run the following command to get information about your current authentication status and user credentials in the Central system:
```

Comment:
I'd say information here, you do not get any credentials but rather metadata such as your name, ID, associated roles:
```suggestion
* Run the following command to get information about your current authentication status and user information in the Central system:
```